packages/eslint-plugin-next/lib/index.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
     'no-duplicate-head': require('./rules/no-duplicate-head'),
     'inline-script-id': require('./rules/inline-script-id'),
     'next-script-for-ga': require('./rules/next-script-for-ga'),
+    'no-script-bi-outside-document': require('./rules/no-script-bi-outside-document'),
```

Comment:
Nit (feel free to ignore): I think it would be nicer to not shorten beforeInteractive to `bi` in the naming here. Maybe something like `no-before-interactive-script-outside-doument`?